### ISSUE TITLE
SyntaxError: Unpaired condition control structure

### ISSUE THREAD

**Issue by stayallive** [Stillat/blade-parser-typescript#112]
This code:

```blade
@if($domains->isNotEmpty() || $customDomains->isNotEmpty())
    <option @if($selectedDomain === $value) selected="selected" @endif>{{ $label }}</option>
@endif
```

Results in a `SyntaxError: Unpaired condition control structure` on the first line.

![afbeelding](https://github.com/user-attachments/assets/db10a193-193e-4252-94aa-6da559c06619)

Screenshot is of a larger block of code, I have tried to make the example as minimal as possible.

I know I could use `@selected` instead of the `@if` but this code is perfectly valid and should (and did not) cause any issues before.

**Comment by JohnathonKoster** [comment]
Thanks for the report! I've tracked this down to the `>` character in `@endif>` causing issues. I've addressed this in 2.1.19

Have a fantastic day!

**Comment by stayallive** [comment]
That was super quick, thanks! 🚀 Just tested and works like a charm again ❤️
